docs/reference/query-dsl/simple-query-string-query.asciidoc

```diff
@@ -273,33 +274,39 @@ GET /_search
 <1> The `subject` field is three times as important as the `message` field.
 
 [[simple-query-string-synonyms]]
-===== Synonyms
+===== Multi-position tokens
 
-The `simple_query_string` query supports multi-terms synonym expansion with the <<analysis-synonym-graph-tokenfilter,
-synonym_graph>> token filter. When this filter is used, the parser creates a phrase query for each multi-terms synonyms.
-For example, the following synonym: `"ny, new york"` would produce:
+The `simple_query_string` query supports
+<<token-graphs-multi-position-tokens,multi-position tokens>> produced by graph
+token filters, such as <<analysis-synonym-graph-tokenfilter,`synonym_graph`>> or
+<<analysis-word-delimiter-graph-tokenfilter,`word_delimiter_graph`>>.
+
+By default, the parser creates a <<query-dsl-match-query-phrase,`match_phrase`>>
+query for each multi-position token. For example, the parser creates a
+`match_phrase` query for the multi-word synonym `ny, new york`:
```

Comment:
I intentionally chose to leave this example largely as is. Synonyms seem to be a more common use case. However, I'm happy to change the example or add another if wanted.